diff --git a/xhprof_html/index.php b/xhprof_html/index.php
index 3629336..135f718 100644
--- a/xhprof_html/index.php
+++ b/xhprof_html/index.php
@@ -33,7 +33,7 @@
 
 // by default assume that xhprof_html & xhprof_lib directories
 // are at the same level.
-$GLOBALS['XHPROF_LIB_ROOT'] = dirname(__FILE__) . '/../xhprof_lib';
+$GLOBALS['XHPROF_LIB_ROOT'] = 'xhprof';
 
 include_once $GLOBALS['XHPROF_LIB_ROOT'].'/display/xhprof.php';
 
